### ISSUE TITLE
Add Bitbucket support

### ISSUE THREAD

**Issue by mrmartineau** [jkup/pullit#17]
I'm sure this means forking your codebase somehow, but Bitbucket support would be amazing :)

**Comment by jkup** [comment]
Yeah this would be awesome. I think we could move the github specific stuff to it's own file and just have an abstraction on top of both APIs.

**Comment by mrmartineau** [comment]
@jkup that sounds like a great idea. it would definitely make it easier to extend to further platforms.
